draft-irtf-cfrg-frost.md

```diff
@@ -855,16 +887,27 @@ The value of the contextString parameter is "FROST-P256-SHA256".
 
 - Group: P-256 (secp256r1) {{x9.62}}
   - Cofactor (`h`): 1
-  - SerializeElement: Implemented using the compressed Elliptic-Curve-Point-to-Octet-String
+  - SerializeNonIdentityElement: Implemented using the compressed Elliptic-Curve-Point-to-Octet-String
     method according to {{SECG}}.
-  - DeserializeElement: Implemented by attempting to deserialize a public key using
+    Additionally, this function validates that the resulting element is not the group
+    identity element.
+  - DeserializeNonIdentityElement: Implemented by attempting to deserialize a public key using
     the compressed Octet-String-to-Elliptic-Curve-Point method according to {{SECG}},
     and then performs partial public-key validation as defined in section 5.6.2.3.4 of
     {{!KEYAGREEMENT=DOI.10.6028/NIST.SP.800-56Ar3}}. This includes checking that the
     coordinates of the resulting point are in the correct range, that the point is on
     the curve, and that the point is not the point at infinity. Additionally, this function
     validates  that the resulting element is not the group identity element.
     If these checks fail, deserialization returns an error.
+  - SerializeElement: Implemented using the compressed Elliptic-Curve-Point-to-Octet-String
+    method according to {{SECG}}.
+  - DeserializeElement: Implemented by attempting to deserialize a public key using
+    the compressed Octet-String-to-Elliptic-Curve-Point method according to {{SECG}},
+    and then performs partial public-key validation as defined in section 5.6.2.3.4 of
+    {{!KEYAGREEMENT=DOI.10.6028/NIST.SP.800-56Ar3}}. This includes checking that the
+    coordinates of the resulting point are in the correct range, that the point is on
+    the curve, and that the point is not the point at infinity. 
+    If these checks fail, deserialization returns an error.
```

Comment:
The P256 thing excludes the point at infinity, which is usually the identity. Is there a way to exclude the identity for this version of P256 at all?

Reply:
P256 is a group, which requires inclusion of the identity element.